_layouts/landing.html

```diff
@@ -119,24 +119,41 @@ <h5 class="title">Reference</h5>
           <div class="row">
             <div class="col-xs-12 col-lg-8">
               <h5 class="title">How do I?</h5>
```

Comment:
Are we still retaining this heading? Wondering if it makes sense to remove this and only retain the tabs

Reply:
Updating in new commit